### DISCUSSION TITLE
Admin dashboard

### DISCUSSION THREAD

**Discussion by mihaeldonko** [microweber/microweber#905] [Q&A]
Hello, 
I am a student working for a small company and we want to use ur program for CMS of a blog on some website, but our client want to have a simpler admin dashboard as we have no need for stuff like modules so i am asking if if there is a way u can edit/hide the unused sections.

Thank you for ur time!
-Mihael

**Comment by peter-mw** [comment]
Hello, you can remove the non needed modules from the uninstall button

![image](https://user-images.githubusercontent.com/5698247/179196610-99c252f8-fd28-48fc-b6f6-4546c673fd9d.png)


If you need the module again you can filter them and install them again

**Reply by Ezyweb-uk** [reply]
Peter, it would be a nice feature if one day the admin navigation menu could be configurable by login profile, so that eg 'editors' would only see and have access to what they need.

**Reply by peter-mw** [reply]
Nice idea @Ezyweb-uk , we will try this when we implement the Roles and Permissions #861

**Reply by mihaeldonko** [reply]
Hey, thanks for the anwser but i didnt mean this configuration. 
[
![image](https://user-images.githubusercontent.com/109346556/179200616-cc7e4ac6-353c-40e7-a4e0-82f7e5c78412.png)
](url)
I meant like this on picture if i can configure and remove this navbar items if they are unused on application.
Thank you again for ur time!

**Reply by peter-mw** [reply]
Hi, for now its not possible to remove by settings, you can remove them manually from the code by editing the file `userfiles/modules/admin/header.php`

**Reply by mihaeldonko** [reply]
Thank you!!! I was looking for that
